### PR TITLE
Fixes to the urdf to mjcf conversion 

### PR DESCRIPTION
**Pull request by alik-git**
Changes:

- Add a "base" body to the root. TODO: how to correctly compute the inertial values

- Correct the DOF caclulation

This should be run with 
```
python urdf2mjcf/convert.py \
              ../sim/sim/resources/gpr/robot_fixed.urdf \
              --output ../sim/sim/resources/gpr/robot_fixed.xml \
              --no-frc-limit \
              --default-position ""0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0""
```

### CODE REVIEW

File: urdf2mjcf/convert.py [hunk 1]
```diff
@@ -205,12 +205,32 @@ def add_root_body(root: ET.Element) -> None:
...
-    # Move existing bodies and geoms under root_body
+    # Create base body
+    base_body = ET.SubElement(
+        root_body,
+        "body",
+        attrib={"name": "base"},
+    )
+
+    # Add inertial properties to base body
+    ET.SubElement(
```
Comment (budzianowski) [inline urdf2mjcf/convert.py]: remove this - this should be automatically computed

File: tests/sample/robot_test.xml [hunk 2]
```diff
@@ -39,79 +39,81 @@
     <body name="root" pos="0 0 0.63" quat="1 0 0 0">
       <freejoint name="root" />
```
Comment (budzianowski) [inline tests/sample/robot_test.xml:40]: this was created with new code but old urdf right?

**Review by budzianowski — APPROVED** [review]

File: tests/sample/robot_test.xml (hunk 2 above)
Comment (alik-git) [reply tests/sample/robot_test.xml:40]: correct. the input was 
```
urdf2mjcf/tests/sample/robot.urdf
```

**Comment by codekansas** [comment]
lgtm

**Pull request merged**